src/sinks/tsd-query-log-sink.ts

```diff
@@ -72,22 +72,25 @@ export class TsdQueryLogSink implements tsdDef.Sink {
     private static stringify(metricsEvent:tsdDef.MetricsEvent):string {
         var transformedMetricsEvent = TsdQueryLogSink.transformMetricsEvent(metricsEvent);
 
-        return JSON.stringify(utils.stenofy(transformedMetricsEvent),
+        return JSON.stringify(utils.stenofy(transformedMetricsEvent, transformedMetricsEvent.annotations["_host"]),
             (key, value) => {
                 if (tsdSink.TsdSink.isMetricSample(value)) {
-                    return {
-                        value: (<tsdDef.MetricSample>value).getValue(),
-                        unit: (<tsdDef.MetricSample>value).getUnit()
+                    // TODO(matthayter): implement unit numerators & denominators
```

Comment:
Argh - the problem is the conflating of units with scales in 2F, which are then separated in 2G. I'd be doing a bunch of throw-away work for no reason (no release of 2F, yeah?). I can't justify that right now. I'm definitely getting units with scales in 2G.

We okay on this?